selftest/global-variable/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+/*
+struct config_t {
+    int a;
+    int b;
+    int c;
+};
+*/
+import "C"
+import (
+	"encoding/binary"
+	"fmt"
+	"os"
+	"syscall"
+	"time"
+	"unsafe"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+)
+
+func exitWithErr(err error) {
+	fmt.Fprintln(os.Stderr, err)
+	os.Exit(-1)
+}
+
+func initGlobalVariable(bpfModule *bpf.Module) {
+	rodata, err := bpfModule.GetMap(".rodata")
```

Comment:
Is there a way you think we can support multiple rodata (for multiple variables) like described at: https://lore.kernel.org/bpf/20211021014404.2635234-8-andrii@kernel.org/ ?

Reply:
I have updated the selftest to cover multiple rodata.